ui/components/app/account-menu/index.scss

```diff
@@ -136,6 +136,7 @@
       margin-top: 5px;
       margin-right: 10px;
       background-color: var(--color-background-alternative);
+      border: 1px solid var(--color-border-default);
       color: var(--color-text-default);
       font-weight: normal;
       letter-spacing: 0.5px;
```

Comment:
Adding border to tag so the background doesn't become invisible on hover

### Before
<img width="398" alt="Screen Shot 2022-05-19 at 3 59 09 PM" src="https://user-images.githubusercontent.com/8112138/169417862-23a3b241-a926-4178-b92e-ea5a63256062.png">


### After
<img width="400" alt="Screen Shot 2022-05-19 at 3 58 18 PM" src="https://user-images.githubusercontent.com/8112138/169417784-1480240f-a234-4dc4-8f98-77af25eeb9a7.png">